.yamllint.yaml

```diff
@@ -0,0 +1,14 @@
+---
+extends: default
+
+rules:
+  line-length:
+    max: 88  # Matches Python line length
```

Comment:
@cbachhuber we could increase python line length there is a huge war what is the best xD.
I would prefer increasing line length sinde I have widescreen monitor